puppet/modules/site_config/manifests/syslog.pp

```diff
@@ -11,4 +11,13 @@
     content => '$ModLoad mmanon
 action(type="mmanon" ipv4.bits="32" mode="rewrite")'
   }
+
+  augeas {
+    'logrotate_leap_deploy':
+      context => '/files/etc/logrotate.d/deploy/rule',
```

Comment:
Would you mind if we made the logrotate.d configuration file called 'leap_deploy', instead of just 'deploy'?